build_runner/test/common/matchers.dart

```diff
@@ -30,6 +30,10 @@ class _AssetGraphMatcher extends Matcher {
     if (graph.allNodes.length != _expected.allNodes.length) matches = false;
     for (var node in graph.allNodes) {
       var expectedNode = _expected.get(node.id);
+      if (node.isDeleted != node.isDeleted) {
```

Comment:
one of these should be `expectedNode.isDeleted` ;)

Reply:
You don't like tautologies?